source/linux/basic/installing_uninstalling_software.rst

```diff
@@ -0,0 +1,76 @@
+Installing and Uninstalling Software
+====================================
```

Comment:
I am not sure if this is really useful. There are so many distributions out there. For my perspective we should concentrate on the things that are relevant in interaction with our boards. This is not.

Reply:
I've added a note and a link to our Yocto manual.